v23.1/drop-type.md

```diff
@@ -33,7 +33,7 @@ The user must be the owner of the type.
 - You cannot drop a type or view that is in use by a table.
```

Comment:
Page summary needs to be updated here, too.

Reply:
Updated